static/app/views/dashboardsV2/widgetBuilder/widgetLibrary/index.tsx

```diff
@@ -38,5 +38,8 @@ export function WidgetLibrary({onWidgetSelect}: Props) {
 const WidgetLibraryWrapper = styled('div')`
   display: flex;
   flex-direction: column;
-  gap: ${space(2)};
+`;
+
+const Header = styled('h5')`
+  margin-left: ${space(2)};
```

Comment:
should the library icons also have a padding of 16px to the left?

Reply:
![image](https://user-images.githubusercontent.com/29228205/155947759-a8fee987-c1de-4a33-a998-007bd817d4aa.png)

Reply:
We talked this morning about the W having one pixel between the margin, causing this shift. We decided to address this later because it is lower priority.